common/config/azure-pipelines/jobs/regression-testing.yaml

```diff
@@ -9,29 +9,29 @@
 trigger: none
 
 schedules:
-- cron: "0 5 * * *"
-  displayName: Daily midnight build
-  branches:
-    include:
-    - master
-    - release/*
+  - cron: "0 5 * * *"
+    displayName: Daily midnight build
+    branches:
+      include:
+        - master
+        - release/*
 
 jobs:
-- template: ci-core.yaml
-  parameters:
-    name: Node_14_x
-    nodeVersion: 14.x
-    pool:
-      demands: Agent.OS -equals $(platform)
-- template: ci-core.yaml
-  parameters:
-    name: Node_10_17
-    nodeVersion: 10.17.0
-    pool:
-      demands: Agent.OS -equals $(platform)
-- template: ci-core.yaml
-  parameters:
-    name: Node_10
-    nodeVersion: 10.x
-    pool:
-      demands: Agent.OS -equals $(platform)
+  - template: ci-core.yaml
+    parameters:
+      name: Node_14_x
+      nodeVersion: 14.x
+      pool:
+        demands: Agent.OS -equals $(platform)
+  # - template: ci-core.yaml
```

Comment:
Why's this commented out?

Reply:
12.17 is currently used as the main CI build version due to the issues with MacOS and Node 14. I'm hoping I can get this resolved by EOW.